src/app/modules/search-config/services/search-global-form-builder/search-global-form-builder.service.ts

```diff
@@ -17,43 +17,76 @@
  * under the License.
  */
 import { Injectable } from '@angular/core';
-import { FormControl, FormGroup } from '@angular/forms';
+import { FormArray, FormControl, FormGroup, Validators } from '@angular/forms';
 import { marker } from '@biesbjerg/ngx-translate-extract-marker';
+import { SearchCollectionComponent } from '@search-config/components/search-collection/search-collection.component';
 import { CollectionService } from '@services/collection-service/collection.service';
+import { CollectionField } from '@services/collection-service/models';
 import { toKeywordOptionsObs, toTextOptionsObs } from '@services/collection-service/tools';
 import { DefaultValuesService } from '@services/default-values/default-values.service';
 import { MainFormService } from '@services/main-form/main-form.service';
-import { ConfigFormGroup, InputFormControl, SelectFormControl, SelectOption, SliderFormControl } from '@shared-models/config-form';
+import { CollectionConfigFormGroup } from '@shared-models/collection-config-form';
+import {
+  ComponentFormControl, ConfigFormGroup, InputFormControl,
+  SelectFormControl, SelectOption, SliderFormControl
+} from '@shared-models/config-form';
 import { Observable } from 'rxjs';
 
-export class SearchGlobalFormGroup extends ConfigFormGroup {
-
+export class SearchCollectionFormGroup extends CollectionConfigFormGroup {
   public constructor(
+    collection: string,
+    collectionService: CollectionService,
     textFieldsObs: Observable<Array<SelectOption>>,
     keywordFieldsObs: Observable<Array<SelectOption>>
   ) {
-    super(
+    super(collection,
       {
-        name: new InputFormControl(
-          null,
-          marker('Placeholder'),
-          marker('Placeholder descritpion'),
-          null,
-          { title: 'Search' }),
+        collection: new SelectFormControl(
+          collection,
+          marker('Collection'),
+          undefined,
+          false,
+          collectionService.getCollections().map(c => ({ label: c, value: c })),
+          {
+            optional: false,
+            resetDependantsOnChange: true
```

Comment:
If my upcoming PR about the collectionService is merged before, you will have to add an option there, otherwise i will

Reply:
You will :)

Reply:
Just merged the PR ;)